lib/raster/Makefile

```diff
@@ -1,6 +1,7 @@
 MODULE_TOPDIR = ../..
 
 LIB = RASTER
+LIBES = $(PARSONLIB)
```

Comment:
`$(PARSONLIB) ` should probably be added instead to:

https://github.com/OSGeo/grass/blob/410cf5ccfd9c8617162ab73411248ddc053fd389/include/Make/Grass.make#L215

in order to assure build order.

Reply:
It seems that the same error is still appearing after applying the suggested fix...

Reply:
With #4671 merged, it will work.

Reply:
Thank you for the help—got some good insights!

Reply:
#4671 is merged, please rebase against main and try again.